src/components/Markdown/Markdown.tsx

```diff
@@ -38,6 +38,10 @@ const handleDetails = (text: string): string | null => {
     return `<blockquote class="${style.details}">${result}</blockquote>`
 }
 
+const handleLogos = (text: string): string | null => {
```

Comment:
The pipeline fails because this function actually returns `string`

![image](https://github.com/user-attachments/assets/a64051e4-ea5b-469f-b8e2-f3b703321cd9)

```suggestion
const handleLogos = (text: string): string => {
```